src/lib/toMmdFlowChart.ts

```diff
@@ -0,0 +1,10 @@
+import { Entity, Rel, Verbs } from "../classes/Entity";
+interface Options {
+  orientation: "LR" | "TB" | "BT" | "RL";
+}
+export default function toMMdFlowChart(root: Entity, options: Options) {
```

Comment:
is `MMd` intentional? are you trying to make a function instead of the class?

Reply:
I thought MMd stands for Mermaid?

Reply:
yes sure but I guess it would be MMD or Mmd, maybe the latter is less shouted